symphony/bdk/core/service/message/message_service.py

```diff
@@ -34,6 +35,38 @@ def __init__(self, messages_api: MultiAttachmentsMessagesApi, message_suppressio
         self._auth_session = auth_session
         self._retry_config = retry_config
 
+    async def send_simple_message(
```

Comment:
would the pythonic way be to have a simple send_message that supports both message as str | Message?

Reply:
It seems pythonic way seems to have all arguments put as optional and implement the behavior based on the passed arguments, see https://stackoverflow.com/a/6434546

Reply:
One of the question I had is whether we allow ourselves to break the backward compatibility of our API. I would say no, so I will add an extra optional argument of type Message on the send_message and blast_message endpoints.

Reply:
as we have not published a 2.0 yet we can right?